app/views/cost_items/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col-md-12">
-    <h1>Cost Items</h1>
+    <h1>Cost types</h1>
```

Comment:
I believe it should be sentence case rather than title case.

I will be creating a new PR to ensure that I have the correct casing across H1 elements as it is currently inconsistent.